js/login.js

```diff
@@ -0,0 +1,123 @@
+const MASTER_ACCOUNT = {
+  EMAIL: 'test@codeit.com',
+  PASSWORD: 'codeit101'
+}
+
+const REGEX = {
+  EMAIL: /^[0-9a-zA-Z]([-_\.]?[0-9a-zA-Z])*@[0-9a-zA-Z]([-_\.]?[0-9a-zA-Z])*\.[a-zA-Z]{2,3}$/i,
+  PASSWORD: '',
+}
+
+const ERROR_MSG = {
+  BLANK: {
+    EMAIL: '이메일을 입력해 주세요.',
+    PASSWORD: '비밀번호를 입력해 주세요.'
+  },
+  INVALID: {
+    EMAIL: '올바른 이메일 주소가 아닙니다.'
+  },
+  INCORRECT:{
+    EMAIL: '이메일을 확인해 주세요.',
+    PASSWORD: '비밀번호를 확인해 주세요.'
+  }
+}
+
+Object.freeze(MASTER_ACCOUNT);
+Object.freeze(REGEX);
+Object.freeze(ERROR_MSG);
+
+const $ = (selector, element = document) => element.querySelector(selector);
+const isMaster = (email, password) => email === MASTER_ACCOUNT['EMAIL'] && password === MASTER_ACCOUNT['PASSWORD'];
+const isEmail = (email) => REGEX['EMAIL'].test(email);
+const isBlank = (e) => e.target.value === '';
+
+// 비어있는 인풋에 에러메시지 추가
+const addErrorMsgToBlankInput = (e) => {
+  const blankInput = e.target;
+  const errorMsg = $('.input-error-msg', blankInput.parentNode);
+  if(isBlank(e)) {
```

Comment:
이런 경우에는 개행을 넣어주는게 좋습니다~
```suggestion
  const errorMsg = $('.input-error-msg', blankInput.parentNode);
  
  if(isBlank(e)) {
```